tools/theme/js/version-select.js

```diff
@@ -0,0 +1,87 @@
+window.addEventListener("DOMContentLoaded", function() {
+  function expandPath(path) {
+    // Get the base directory components.
+    var expanded = window.location.pathname.split("/");
```

Comment:
Nit, use `let` instead of `var`

Reply:
Most of this file was copied from the `mike` theme stuff